Classes/Systems/AppRouter/AppController.swift

```diff
@@ -33,6 +33,10 @@ RouterPropsSource {
         sessionManager.addListener(listener: self)
     }
 
+    func getAppClient() -> GithubClient? {
```

Comment:
This feels a bit weird to me... think it would be nice if we can keep the reading logic in one place.

Reply:
No problem so how should I get the app client in the extension?

Reply:
@BasThomas

Reply:
Hm, that's a question I don't know I can answer :(

Reply:
I can make it public or create a getter, or I could move the extension into the main class

Reply:
I feel like option 3 would be the best for now.